File: SecuJournal/sjPassword.cs
using System;
using System.Collections.Generic;
using System.ComponentModel;
using System.Data;
using System.Drawing;
using System.Linq;
using System.Text;
using System.Threading.Tasks;
using System.Windows.Forms;

namespace SecuJournal
{
    public partial class sjPassword : Form
    {
        public sjPassword()
        {
            InitializeComponent();
        }

        private void OKButton_Click(object sender, EventArgs e)
        {
            if (Properties.Settings.Default.SJPassword == TextBox1.Text)
            {
                this.Close();
            }
            else
            {
                MessageBox.Show("You have entered the wrong password for this journal.", "Journal Password", MessageBoxButtons.OK, MessageBoxIcon.Error);
            }
        }

        private void CancelButton_Click(object sender, EventArgs e)
        {
            Application.Exit();
        }
    }
}

File: SecuJournal/SJPassword.vb
Public Class SJPassword
    Private Sub CancelButton_Click(sender As Object, e As EventArgs) Handles CancelButton.Click
        End
    End Sub

    Private Sub OKButton_Click(sender As Object, e As EventArgs) Handles OKButton.Click
        If My.Settings.SJPassword = TextBox1.Text Then
            Me.Close()
        Else
            MsgBox("You have entered the wrong password for this journal.", MsgBoxStyle.Critical, "Journal Password")
        End If
    End Sub
End Class